haskell/test/Ligo/SegCFMM/Contract.hs

```diff
@@ -0,0 +1,16 @@
+-- SPDX-FileCopyrightText: 2021 Arthur Breitman
+-- SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+-- | LIGO version of the contract.
+module Ligo.SegCFMM.Contract
```

Comment:
There is no other version of the contract.

Likewise, I'd remove `Ligo` from the path/module names as it's not necessary here.

Reply:
Removed the `Ligo` path/name.